apps/www/components/image-with-blur.tsx

```diff
@@ -0,0 +1,14 @@
+import Image, { type ImageProps } from "next/image";
+import React from "react";
+
+type ImageWithBlurProps = ImageProps;
+
+export default function ImageWithBlur(props: ImageWithBlurProps) {
```

Comment:
no default exports please
```suggestion
export function ImageWithBlur(props: ImageWithBlurProps) {
```